test/server/cards/02_SHD/upgrades/BrutalTraditions.spec.ts

```diff
@@ -0,0 +1,48 @@
+describe('Brutal Traditions', function() {
+    integration(function(contextRef) {
+        describe('Brutal Tradition\'s ability', function() {
```

Comment:
Can we add a test that ensures the Brutal Tradition can be played from discard multiple times per phase?

Reply:
added the test!